project-base/src/Shopsys/ShopBundle/Resources/styles/front/common/core/base.less

```diff
@@ -69,124 +69,124 @@ h1, h2, h3, h4, h5, h6 {
 h1 {
     margin: 0 0 10px 0;
 
-    font-size: 20px;
+    font-size: 29px;
     font-weight: normal;
 
     @media @query-md {
-        font-size: 24px;
+        font-size: 31px;
     }
 
     @media @query-lg {
         margin-bottom: 15px;
 
-        font-size: 28px;
+        font-size: 34px;
     }
 
     @media @query-vl {
         margin-bottom: 20px;
 
-        font-size: 32px;
+        font-size: 36px;
     }
 }
 
 h2 {
     margin: 0 0 10px 0;
 
-    font-size: 16px;
+    font-size: 28px;
     font-weight: normal;
 
     @media @query-md {
-        font-size: 18px;
+        font-size: 29px;
     }
 
     @media @query-lg {
         margin-bottom: 15px;
 
-        font-size: 20px;
+        font-size: 29px;
     }
 
     @media @query-vl {
         margin-bottom: 20px;
 
-        font-size: 22px;
+        font-size: 32px;
     }
 }
 
 h3 {
     margin: 0 0 10px 0;
 
-    font-size: 14px;
+    font-size: 21px;
     font-weight: normal;
 
     @media @query-md {
-        font-size: 15px;
+        font-size: 20px;
```

Comment:
is this OK, everywhere there is bigger size for `@query-md` except this one and in the state before it was also bigger
```
    font-size: 14px;	    font-size: 21px;
    font-weight: normal;	    font-weight: normal;
     @media @query-md {	    @media @query-md {
        font-size: 15px;
```

Reply:
Thanks for the point. Now it is changed. Please review my code now.